### ISSUE TITLE
Web-app setup into subdomain

### ISSUE THREAD

**Issue by ghost** [signalpoint/DrupalGap#865]
May can i setup a web-app into separate or subdomain?

**Comment by signalpoint** [comment]
@tazimhossain You'll need to enable CORS for Drupal.

**Comment by ghost** [comment]
I have enabled CORS and connect successfully, but when I enter username and password, its not login.
any idea?

**Comment by signalpoint** [comment]
@tazimhossain You need to use Chrome's developer tools and monitor the JavaScript console log, and the Network tab for more information. Please report relevant log messages, and network status information here for further assistance.

**Comment by ghost** [comment]
@signalpoint thanx for quick reply. I have no error, but when I try to login after disable chrome cors security, i can login fine. But, when i trying to login normally- its not login.

 I attached an image with login and network status.
![screenshot-override](https://cloud.githubusercontent.com/assets/7555986/19227903/6515d9c6-8ee1-11e6-9222-50e969965ff5.png)

For inform you.
 I added this codes into my conf file for avoid cors problem
`Header always set Access-Control-Allow-Origin "*"`
`Header always set Access-Control-Allow-Methods "POST, GET, OPTIONS, DELETE, PUT"`
`Header always set Access-Control-Allow-Headers "Content-Type, X-CSRF-Token, origin, accept"`

I skip this "http://docs.drupalgap.org/7/Developer_Guide/CORS" step.

 Sorry for hiding my domain name... :)

**Comment by ghost** [comment]
Hey @signalpoint, any ideas? Please, i need help.

**Comment by signalpoint** [comment]
@tazimhossain Nope, sorry. This is why I've always placed apps in the same domain as the Drupal site.

**Comment by ghost** [comment]
@signalpoint, 
Hmm... ok...

One more question, have any way to develop drupalgap app without jquery mobile? I like to use angular to develop my apps.

 thank you. :)

**Comment by signalpoint** [comment]
@tazimhossain DrupalGap 7 no, DrupalGap 8 yes.

DG8 is itself headless and let's you pick frameworks to put on top of it, e.g. Angular, Bootstrap, etc. : http://demo.drupalgap.org/8/

jDrupal however is great for Drupal 7 and Drupal 8 and any JavaScript/CSS frameworks: http://jdrupal.tfrankenstein.com/

**Comment by ghost** [comment]
@signalpoint how r u?
Is subdomain login support with drupal 8?

**Comment by signalpoint** [comment]
@tazimhossain Subdomain support is possible on Drupal 7 and 8. You just need to configure the CORS policy properly on your server, and within your app's index.html file. Although I have never tried either in a subdomain, but I know others have done it.

**Comment by ghost** [comment]
@signalpoint sorry for ask you again. I am trying hard for doing support subdomain authentication for my projects. But, i can't figure it out. Can you please help me to do that? or any docs for solve my problem?

Thanx. :(

**Comment by signalpoint** [comment]
@tazimhossain There are no docs on this issue as far as I know. People have tried the sub domain approach, I never had any luck with it, many have asked about it, etc, so there will be many issues in the various issue queues talking about the situation. Maybe try the "cookie domain" technique that I think is available via a settings.php config.
